content/cli/workspaces.md

```diff
@@ -283,29 +284,34 @@ module.exports = function(options) {
     ...options,
     externals: [],
   };
-}
+};
 ```
 
 #### Assets
 
-TypeScript compilation automatically distributes compiler output (`.js` and `.d.ts` files) to the specified output directory.  It can also be convenient to distribute non-TypeScript files, such as `.graphql` files, `images`, `.html` files and other assets.  This allows you to treat `nest build` (and any initial compilation step) as a lightweight **development build** step, where you may be editing non-TypeScript files and iteratively compiling and testing.
+TypeScript compilation automatically distributes compiler output (`.js` and `.d.ts` files) to the specified output directory. It can also be convenient to distribute non-TypeScript files, such as `.graphql` files, `images`, `.html` files and other assets. This allows you to treat `nest build` (and any initial compilation step) as a lightweight **development build** step, where you may be editing non-TypeScript files and iteratively compiling and testing.
 
-The value of the `assets` key should be an array of elements specifying the files to be distributed.  The elements can be simple strings with `glob`-like file specs, for example:
+The value of the `assets` key should be an array of elements specifying the files to be distributed. The elements can be simple strings with `glob`-like file specs, for example:
 
 ```typescript
-"assets": ["**/*.graphql"]
+"assets": ["**/*.graphql"],
+"watchAssets": true,
 ```
 
+> info **Hint** To watch your assets during development, use `watchAssets` option
```

Comment:
Is this hint really needed with the additions above?  If we feel it is, I suggest this change:
> info **Hint** To watch your assets during development, use `watchAssets` option
->
> info **Hint** To enable watch mode (live-reload) for assets during development, use the `watchAssets` option (see [global compiler options](/cli/monorepo#global-compiler-options) above for more details).

Reply:
This is what I want to say from above comment that there are multiple way to define assets in `compilerOptions` object or in the `assets` object itself. But I am not good to explain it. I want to hear from you.
```
{
  ...
  "compilerOptions": {
    "assets": [     
       { 
         "include": "**/*.graphql", 
         "exclude": "**/omitted.graphql", 
         "watchAssets": false <-- can use here for specific asset
       },
    ],
    "watchAssets": true <-- can use here as parent
  }
}

```

Reply:
already updated.